ruby/audio/wasapi.cpp

```diff
@@ -188,7 +193,7 @@ struct AudioWASAPI : AudioDriver {
       waveFormat = *(WAVEFORMATEXTENSIBLE*)waveFormatEx;
       CoTaskMemFree(waveFormatEx);
       if(self.audioClient->GetDevicePeriod(&self.devicePeriod, nullptr)) return false;
-      auto latency = max(self.devicePeriod, (REFERENCE_TIME)self.latency * 10'000);  //1ms to 100ns units
+      auto latency = max(self.devicePeriod * 2, (REFERENCE_TIME)self.latency * 10'000);  //1ms to 100ns units
```

Comment:
The change to this line is motivated by an abundance of caution rather than an observed need. On my machine, the default period is 10 ms and WASAPI won't let me set a buffer duration less than 22 ms. In principle, because shared streams only have a single buffer, it wouldn't make much sense to even allow the buffer duration to be less than double the period. I don't know for sure though if that's enforced by WASAPI, and thus the change here.